oeps/architectural-decisions/oep-0058-arch-translations-management.rst

```diff
@@ -0,0 +1,259 @@
+can increase the reach of the Transifex Memory feature to help translate similar strings
+across the entire code/documentation base.
+
+Impacts
+*******
+
+Impact on Translators
+=====================
+
+As we approach the end of the translation upgrade process, we will need to tactically
+move from multiple transifex projects to a single project.  This will require
+coordination with our translators to ensure that moving forward they are providing
+translations in the right place.
+
+Impact on Site Operators
+========================
+
+Currently the translation files for any given service or library is stored at the same
+place as the code, which has generally simplified the deployment story in the past. With
+this change, the translations files will move to their own repository. As we deprecate
+the old translations files, the relevant deployment tooling will need to be updated to
+pull down the translations from the new repository as a part of the deployment process.
+This will impact both the old Ansible based tooling as well as any new docker based
+tooling.
+
+Impact on Developers
+====================
+
+While it won’t directly impact the day-to-day workflow of developers, due to the same
+reasons that we impact site operators (new translations location), we will have to update
+development tools as well. In addion, we will create new instructions for developers on
+how to enable translations for a new service/repo when it comes online.
+
+Final State
+***********
+
+Repositories that generate translation files will have their translation files generated
+and committed via a pull request to the openedx-translation repository via a github
+workflow. Once the translation files from edx-platform and other repositories are moved
+to the openedx-translations repository, the `Transifex GitHub App`_ will link a Transifex
+project of a name such as "Open edX Translations" to the openedx-translations repository.
+A `Transifex GitHub Integration configuration file`_ naming the files that are to be
+translated and the trigger that pulls translation files back into will be created in the
+openedx/translations repository. This link will allow for the `Transifex GitHub App`_ to
+automatically manage the push/pull of the translation files without the need for human
+intervention.
+
+When it comes time to cut an Open Release, a new branch will be formed in the
+openedx-translations repository for this release. This new branch will correspond with
+other release branches such as those found in edx-platform. It is my recommendation that
+a tool in the form of a python library is written to enable the placement of the
+translation files kept in openedx-translations into the repositories the translation
+files are formed from. This tool will manage the placement of translation files through
+an editable translation tool configuration file kept in the repositories that have
```

Comment:
You say "editable translation tool configuration file" a few times in a row. Could we give it a short, succinct name that is easier to read and more memorable? eg the first time you could say, "an editable translation tool configuration file (`translation-config.yml`) and refer to it as such in the next few sentences?

Also, why does "tool" need to be in the name?